proxy/envoy/testdata/envoy-ssl-context.json.golden

```diff
@@ -0,0 +1,297 @@
+{
+  "listeners": [
+    {
+      "port": 80,
```

Comment:
You need to test CDS to get the artifact. I don't think you should enable auth for TCP yet and the config only contains TCP clusters.